### PR TITLE
FND-306 - The domain of of several properties should not be 'Reference', which is too narrow

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Removed 'Reference' as the domain of several properties; eliminated a redundant 'evaluates' property from Analytics that was not used anywhere; adjusted definitions for a few properties in Relations that were not ISO 704 compliant

Fixes: #1095 / [FND-306](https://jira.edmcouncil.org/browse/FND-306)

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](../CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](../ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

File: FND/Relations/Relations.rdf [hunk 1]
```diff
@@ -81,8 +81,7 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;appliesTo">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
 		<rdfs:label>applies to</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
-		<skos:definition>a relation indicating something that is pertinent or relevant to the concept</skos:definition>
+		<skos:definition>is pertinent, suitable, or relevant to</skos:definition>
```
Comment (rivettp) [inline FND/Relations/Relations.rdf:84]: too broad compared to its actual usage in FIBO

File: FND/Relations/Relations.rdf [hunk 2]
```diff
@@ -103,8 +102,7 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;characterizes">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
 		<rdfs:label>characterizes</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
-		<skos:definition>a feature or quality that distinguishes something from something else</skos:definition>
+		<skos:definition>describes the nature, features or quality of</skos:definition>
```
Comment (rivettp) [inline FND/Relations/Relations.rdf:105]: Too broad to be useful. The giveaway is the "or" in the definition - maybe we should have a hygiene test for that!

It's impossible from these definitions to decide whether to use appliesTo or characterizes. Seriously.
We have Catalog characterizes Product; why would it not be appliesTo?
Given that characterizes is used with Aspect (which is at least coherent) then, simce Catalog is nothing at all like an Aspect, that Catalog usage is wrong.

qualifies is subPropertyOf characterizes but with quite different, rather than more specific meaning "limits, constrains, or refines". Again not coherent.

Another usage is that a specific CodeScheme characterizes a CodeRepository. I wouldn't say that it "describes the quality of" it. Again not coherent.

File: FND/Relations/Relations.rdf [hunk 3]
```diff
@@ -132,14 +130,13 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
...
 		<rdfs:label>defines</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-rel-rel;Reference"/>
 		<skos:definition>determines or identifies the essential qualities or meaning of, discovers and sets forth the meaning of, fixes or marks the limits of, demarcates</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;designates">
 		<rdfs:label>designates</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-aap-agt;AutonomousAgent"/>
-		<skos:definition>to name something officially or appoint someone to a position officially</skos:definition>
+		<skos:definition>names something officially or appoints someone to a position officially</skos:definition>
```
Comment (rivettp) [inline FND/Relations/Relations.rdf:139]: No no no.
These are entirely different semantics. They might use the same English word but must not use the same semantic property.

File: FND/Utilities/Analytics.rdf [hunk 4]
```diff
@@ -101,6 +101,14 @@
 		<owl:equivalentClass rdf:resource="&fibo-fnd-utl-alx;Ratio"/>
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;characterizes">
+		<rdfs:domain rdf:resource="&fibo-fnd-utl-alx;Aspect"/>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-rel-rel;isCharacterizedBy">
+		<rdfs:range rdf:resource="&fibo-fnd-utl-alx;Aspect"/>
```
Comment (rivettp) [inline FND/Utilities/Analytics.rdf:109]: we have AddressComponent as a subclass of Aspect, which I think is wrong: these restrictions do not work at all well for that.

File: FND/Relations/Relations.rdf (hunk 1 above)
Comment (ElisaKendall) [reply FND/Relations/Relations.rdf:84]: There are actually 77 uses of the property appliesTo in FIBO.  Some of those uses can be replaced with a more specific property, I agree, but I don't know if there is a replacement that is more specific available for every one of them.  I would rather not make a change to this until we have (1) replaced some of its uses with appropriately more specific properties, and (2) done further research on the remainder to decide whether there is a better definition that could reflect its usage for all of them.  The main change here was to eliminate "a relation indicating something that", which is overly verbose and doesn't meet our ISO 704 compliant approach to definitions.  I didn't change the meaning at all.

File: FND/Relations/Relations.rdf (hunk 2 above)
Comment (ElisaKendall) [reply FND/Relations/Relations.rdf:105]: It is not too broad to be useful - that's the definition of the term 'characterizes'.  I agree that some of the places where it is used should be corrected, such as in the case of a catalog, and will make that change.  Applies to is fine for that case and for the code scheme case, which I will revise accordingly.

File: FND/Relations/Relations.rdf (hunk 3 above)
Comment (ElisaKendall) [reply FND/Relations/Relations.rdf:139]: Ok - I've limited this to naming something offucially, and appoints, as a subproperty makes more sense now

File: FND/Utilities/Analytics.rdf (hunk 4 above)
Comment (ElisaKendall) [reply FND/Utilities/Analytics.rdf:109]: fixed - made it a constituent instead

File: FND/Relations/Relations.rdf (hunk 1 above)
Comment (rivettp) [reply FND/Relations/Relations.rdf:84]: My problem here is less with the property itself, but that the proposed definition does not reflect how it's  used, and should be used.

File: FND/Relations/Relations.rdf (hunk 3 above)
Comment (rivettp) [reply FND/Relations/Relations.rdf]: No, how is "appoints" a subproperty of "names something officially"? An example of the latter might be to name the UK as "The United Kingdom of Grate Britain and Northern Ireland". 
Another reason it does not work for naming is that, if tracking the Authorizer (which this does as the domain of the property), and the object of the example triple is the country, then there seems no way to represent the official name that has been designated.

The other alternative "appoints someone" would make more sense for consistency, since the only uses of designates are as super property of a) appoints and b) delegatesControlTo.

My main concern with using "designates" in this way is that we use "designation" inconsistently to mean both "appointment" (consistent) and naming (inconsistent) - for example we have property preferredDesignation.
In summary I think we should stick to something like "appoints" here and remove the naming-related uses of "designation" as a separate issue.

File: FND/Relations/Relations.rdf (hunk 2 above)
Comment (ElisaKendall) [reply FND/Relations/Relations.rdf]: Raised FND-311 to review uses of appliesTo; added FND-312 to replace hasDescription with isDescribedBy

File: FND/Relations/Relations.rdf (hunk 3 above)
Comment (ElisaKendall) [reply FND/Relations/Relations.rdf]: Ok - I will revise the definition of designates to be appoints someone, and then eliminate the subproperty 'appoints', since it is redundant given this change.
